demo.md

```diff
@@ -38,6 +38,13 @@ $ kubectl get services --namespace kube-system
 
 Note: service namespace should correspond to namespace chosen during argo installation (kube-system is default namespace).
 
+Note: If you are installing Argo on Minikube, you won't get an external IP on creating the service above. Instead, it will just show `pending`. It should look something like below:
+```
+argo-ui                LoadBalancer   <redacted-IP>   <pending>     80:31185/TCP    1h
+```
+
+So, to access the Argo UI, you need to hit the IP of the Docker environment and the internal port. For instance, in the above case, it would be `192.168.99.100:31185`.
```

Comment:
Just FYI, we recently changed the installer to also create the K8s service for argo-ui with type ClusterIP (the release with this change has not been cut yet). When we release the next version, I'll be tweaking the messaging to explain how to access the UI (either via port-forwarding, or changing the service type to LoadBalancer).